css/properties/aspect-ratio.json

```diff
@@ -59,7 +59,9 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": false
+              "version_added": "12.0",
+              "partial_implementation": true,
+              "notes": "Samsung Internet 79 adds internal support only for mapped values"
```

Comment:
The version number here is wrong. I guess the script copies notes and replaced the browser name, but not version numbers?